gemrb/GUIScripts/iwd2/MessageWindow.py

```diff
@@ -109,10 +106,8 @@ def UpdateControlStatus():
 			Button.SetFlags(IE_GUI_BUTTON_NO_IMAGE, OP_SET)
 		else:
 			Button.SetPicture(Portrait, "NOPORTSM")
-	else:
-		GUICommon.GameControl.SetStatus(IE_GUI_CONTROL_FOCUSED)
```

Comment:
What _was_ that? `GameControl` is something now, but not what it was supposed to be ...